src/js/administration/components/Api.tsx

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+import { useMutation, useQueryClient } from "react-query";
+import { ExternalLink } from "../../base";
+import { updateSettings } from "../api";
+import { settingsQueryKeys } from "../querys";
+import { SettingsCheckbox } from "./SettingsCheckbox";
+
+/**
+ * Displays the API settings and allows the users to toggle API access for clients
```

Comment:
Not 100% sure on this docstring. Would probably focus it on the functionality of turning external API access on and off